app/src/docs/v1.api-spec.yaml

```diff
@@ -876,16 +876,99 @@ components:
           items:
             $ref: "#/components/schemas/Role"
           description: An array of Common Service roles the client currently has
-    KcClients:
+    KcClientDetailFull:
       type: object
-      required:
-        - clients
+      description: "Keycloak's representation of a client. see: https://www.keycloak.org/docs-api/5.0/rest-api/index.html#_clientrepresentation"
       properties:
-        clients:
+        access:
+          type: object
+          example: { view: true, configure: true, manage: true }
+        attributes:
+          type: object
+          example: {
+          'saml.assertion.signature': 'false',
+          'saml.multivalued.roles': 'false',
+          'saml.force.post.binding': 'false',
+          'saml.encrypt': 'false',
+          'saml.server.signature': 'false',
+          'saml.server.signature.keyinfo.ext': 'false',
+          'exclude.session.state.from.auth.response': 'false',
+          'saml_force_name_id_format': 'false',
+          'saml.client.signature': 'false',
+          'tls.client.certificate.bound.access.tokens': 'false',
+          'saml.authnstatement': 'false',
+          'display.on.consent.screen': 'false',
+          'saml.onetimeuse.condition': 'false'
+        }
```

Comment:
While this is technically valid as YAML allows JSON objects, we should try to ensure that this is represented as pure YAML instead. I don't know if it needs to be updated so that saml is its own attribute instead, but I suspect you just don't need the curly braces:

```yaml
example:
  saml.assertion.signature: 'false'
  saml.multivalued.roles: 'false'
  saml.force.post.binding: 'false'
  saml.encrypt: 'false'
  saml.server.signature: 'false'
  saml.server.signature.keyinfo.ext: 'false'
  exclude.session.state.from.auth.response: 'false'
  saml_force_name_id_format: 'false'
  saml.client.signature: 'false'
  tls.client.certificate.bound.access.tokens: 'false'
  saml.authnstatement: 'false'
  display.on.consent.screen: 'false'
  saml.onetimeuse.condition: 'false'
```